lib/entities/state/attributes/PresetSelectionStateAttribute.js

```diff
@@ -37,11 +37,10 @@ PresetSelectionStateAttribute.TYPE = Object.freeze({
 });
 
 /**
- *  @typedef {string} PresetSelectionStateAttributeValue
  *  @enum {string}
  *
  */
-PresetSelectionStateAttribute.VALUE = Object.freeze({
+PresetSelectionStateAttribute.INTENSITY = Object.freeze({
```

Comment:
This is so that later we can add, for example, `.MOVEMENT_MODE` and keep it separate from intensity presets.